app/Helpers/database/user-auth.php

```diff
@@ -162,17 +134,20 @@ function changePassword(string $user, string $pass): bool
 
 function hashPassword(string $pass): string
 {
-    return password_hash($pass, PASSWORD_ARGON2ID, [
+    $algorithm = defined('PASSWORD_ARGON2ID') ? PASSWORD_ARGON2ID : PASSWORD_DEFAULT;
```

Comment:
very nice - this solves the potentially missing argon support in php in some local setups.

I'll probably refactor this to use Laravel's capabilities at some point, should be fairly easy to make this compatible.